tests/helpers.nim

```diff
@@ -8,41 +8,29 @@ import unittest2
 export unittest2
 
 const
-  StreamTransportTrackerName = "stream.transport"
-  StreamServerTrackerName = "stream.server"
   DgramTransportTrackerName = "datagram.transport"
+  UdpTransportTrackerName = "webrtc.udp.transport"
+  StunTransportTrackerName = "webrtc.stun.transport"
+  StunConnectionTrackerName = "webrtc.stun.connection"
+
 
   trackerNames = [
-    StreamTransportTrackerName,
-    StreamServerTrackerName,
     DgramTransportTrackerName,
+    UdpTransportTrackerName,
+    StunTransportTrackerName,
+    StunConnectionTrackerName,
   ]
 
-template asyncTest*(name: string, body: untyped): untyped =
-  test name:
-    waitFor((proc () {.async, gcsafe.} = body)())
```

Comment:
why has this been removed?

Reply:
Because it's already defined here : https://github.com/vacp2p/nim-webrtc/blob/master/tests/asyncunit.nim#L19-L24

Reply:
I made the change. I now use the chronos asyncTest.